build/pipelines/templates/build-console-compliance-job.yml

```diff
@@ -146,7 +148,7 @@ jobs:
     continueOnError: true
 
   # Set XES_SERIALPOSTBUILDREADY to run Security and Compliance task once per build
-  - powershell: Write-Host “##vso[task.setvariable variable=XES_SERIALPOSTBUILDREADY;]true”  
+  - powershell: Write-Host "##vso[task.setvariable variable=XES_SERIALPOSTBUILDREADY;]true"

```

Comment:
Those quotes were a bit surprising. I don't think that was intentional was it? I took the liberty to fix them...